### PR TITLE
Add rubygem-foreman_inventory_upload package

### PR DESCRIPTION
**Pull request by ShimShtein**
Supported Versions:

 * Nightly
 * 1.23
 * 1.22
 * 1.21
 * 1.20

### CODE REVIEW

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -81,6 +81,7 @@
       <packagereq type="default">tfm-rubygem-fog-xenserver</packagereq>
       <packagereq type="default">tfm-rubygem-fog-xml</packagereq>
       <packagereq type="default">tfm-rubygem-font-awesome-sass</packagereq>
+      <packagereq type="default">tfm-rubygem-foreman_inventory_upload</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml]: I think this should be in `comps-katello-server-rhel7.xml`

**Comment by ShimShtein** [comment]
@evgeni, moved to `comps-katello`

File: comps/comps-katello-server-rhel7.xml [hunk 2]
```diff
@@ -71,6 +71,9 @@
 
       <!-- other katello related packages -->
       <packagereq type="default">rhel8-kickstart-setup</packagereq>
+
+      <!-- Additional plugins dependent on katello -->
+      <packagereq type="default">tfm-rubygem-foreman_inventory_upload</packagereq>
```
Comment (ekohl) [inline comps/comps-katello-server-rhel7.xml]: This should be under thirdparty I think. We already have foreman_scc_manager there which is another plugin dependent on katello.

File: comps/comps-katello-server-rhel7.xml [hunk 3]
```diff
@@ -79,6 +82,7 @@
     <description>Katello Server Documentation Packages</description>
     <uservisible>true</uservisible>
     <packagelist>
+      <packagereq type="default">tfm-rubygem-foreman_inventory_upload-doc</packagereq>
```
Comment (ekohl) [inline comps/comps-katello-server-rhel7.xml]: Please keep this consistent with the rest and set it to optional.

**Comment by ShimShtein** [comment]
@ekohl done!

File: packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec [hunk 4]
```diff
@@ -0,0 +1,105 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
```
[the remark this answers is not in the document]
Comment (ShimShtein) [reply packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: Done again :)

Sorry about that, I thought I removed the lines.

File: packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+%global plugin_name inventory_upload
+%global foreman_min_version 1.20
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1.dev1
+Release: 1%{?foremandist}%{?dist}
+Summary: Adds ability to upload hosts managed in Foreman to Red Hat cloud inventory
+Group: Applications/Systems
+License: GPLv3
+URL: https://github.com/theforeman/foreman_inventory_upload
```
Comment (ekohl) [inline packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: Not blocking the merge, but when do you plan to transfer the repo?

Comment (ShimShtein) [reply packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec]: In a near future, I assume in the next couple of days

**Pull request merged**